meta-edison-bsp/classes/image_types_edison.bbclass

```diff
@@ -107,6 +107,7 @@ IMAGE_CMD_toflash () {
 
 	# update image name inside flashall.sh
 	sed -e "s/^IMAGE_NAME=.\+$/IMAGE_NAME=\"${IMAGE_BASENAME}\"/" -i ${WORKDIR}/toFlash/flashall.sh
+	sed -e "s/^IMAGE_NAME=.\+$/IMAGE_NAME=${IMAGE_BASENAME}/" -i ${WORKDIR}/toFlash/flashall.bat
```

Comment:
OK thanks, stupid copy-paste error. I'll fix it and test this on Windows once the builds finishes.